c++/Makefile.am

```diff
@@ -576,6 +576,14 @@ capnp_evolution_test_SOURCES = src/capnp/compiler/evolution-test.c++
 
 capnp_afl_testcase_LDADD = libcapnp-test.a libcapnp-rpc.la libcapnp.la libkj.la libkj-async.la
 capnp_afl_testcase_SOURCES = src/capnp/afl-testcase.c++
+
+fuzz_capnp_run_LDADD = libcapnp-test.a libcapnp-rpc.la libcapnp.la libkj.la libkj-async.la
+fuzz_capnp_run_SOURCES = src/capnp/fuzz-run.c++
+if HAS_NO_FUZZING_ENGINE
+    fuzz_capnp_run_SOURCES += src/capnp/fuzz-driver.c++
```

Comment:
What's the purpose of this fuzz driver exactly? Maybe we should just not build the executable at all if a fuzz driver wasn't specified?

Reply:
The main purpose is for CI to check compilation of the fuzz targets, without requiring a real/complex fuzz driver

Reply:
Why not actually install the LLVM fuzzer in CI?

Reply:
Trying that with the latest commit